UM/Qt/Bindings/MainWindow.py

```diff
@@ -89,7 +90,9 @@ def __init__(self, parent = None):
                                       self.DEFAULT_WINDOW_HEIGHT)
 
         self.setGeometry(restored_geometry)
-        self.setWindowState(int(self._preferences.getValue("general/window_state")))
+        # self.setWindowState(int(self._preferences.getValue("general/window_state")))
```

Comment:
Not sure why this was commented out. There is still a 'setWindowState' in Qt 6, and the enums used are still the same.

It may be something that was removed in 6.x and added back in 6.2?

Reply:
Ah,we should just cast it to the enum I think, instead of an int.